support/api-testsupport/pom.xml

```diff
@@ -51,4 +51,17 @@
     </dependency>
 
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Some of our users are declaring dependencies to styx-api-testsupport, can we remove the deployment for it? Also both styx-api-testsupport and styx-testsupport are referenced in the BOM file.

Reply:
Ok. I'll review the styx-api-testsupport, and remove the two others from the BOM.

Reply:
Hi @dvlato. I have put back styx-api-testsupport for now. Although it ought to go in future. I have also edited the BOM accordingly.